packages/eslint-plugin/src/rules/no-empty-interface.ts

```diff
@@ -84,6 +84,7 @@ export default util.createRule<Options, MessageIds>({
 
             const isInAmbientDeclaration = !!(
               util.isDefinitionFile(filename) &&
+              // eslint-disable-next-line @typescript-eslint/no-unsafe-enum-comparison
               scope.type === 'tsModule' &&
```

Comment:
```suggestion
              scope.type === ScopeType.tsModule &&
```

this is a legit error